manifests/m/mulaRahul/Keyviz/1.0.6/mulaRahul.Keyviz.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: mulaRahul.Keyviz
+PackageVersion: 1.0.6
+InstallerType: inno
+Scope: user
+UpgradeBehavior: install
+ReleaseDate: 2022-08-28
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
Not sure why the validation pipeline didn't catch this. The installer installs fine, but the program requires VCRedist 2015 to launch. 
![image](https://user-images.githubusercontent.com/114952565/208179731-97c15403-062a-4517-9ac1-19913b9927b8.png)